content/trusted-content/official-images/contributing.md

```diff
@@ -0,0 +1,55 @@
+  This article describes how Docker Official Images are created,
+  and how you can contribute or leave feedback.
+keywords: docker official images, doi, contributing, upstream, open source
+---
+
+Docker, Inc. sponsors a dedicated team that's responsible for reviewing and
+publishing all content in Docker Official Images. This team works in
+collaboration with upstream software maintainers, security experts, and the
+broader Docker community.
+
+While it's preferable to have upstream software authors maintaining their
+Docker Official Images, this isn't a strict requirement. Creating
+and maintaining images for Docker Official Images is a collaborative process.
+It takes place [openly on GitHub](https://github.com/docker-library/official-images)
+where participation is encouraged. Anyone can provide feedback, contribute
+code, suggest process changes, or even propose a new Official Image.
+
+> **Note**
+>
+> Docker Official Images are the intellectual property of Docker.
+
+## Creating a Docker Official Image
+
+From a high level, an Official Image starts out as a proposal in the form
+of a set of GitHub pull requests. The following GitHub repositories detail the proposal requirements:
+
+- [Docker Official Images repository on GitHub](https://github.com/docker-library/official-images#readme)
+- [Documentation for Docker Official Images](https://github.com/docker-library/docs#readme)
+
+The Docker Official Images team, with help from community contributors, formally
+review each proposal and provide feedback to the author. This initial review
+process may require a bit of back-and-forth before the proposal is accepted.
+
+There are subjective considerations during the review process. These
+subjective concerns boil down to the basic question: "is this image generally
+useful?" For example, the [Python](https://hub.docker.com/_/python/)
+Docker Official Image is "generally useful" to the larger Python developer
+community, whereas an obscure text adventure game written in Python last week is
+not.
+
+Once a new proposal is accepted, the author is responsible for keeping
+their images up-to-date and responding to user feedback. The Official
+Repositories team becomes responsible for publishing the images and
+documentation on Docker Hub. Updates to the Docker Official Image follow the same pull request process, though with less review. The Docker Official Images team ultimately acts as a gatekeeper for all changes, which helps mitigate the risk of quality and security issues from being introduced.
+
+## Submitting feedback for Docker Official Images
+
+All Docker Official Images contain a **User Feedback** section in their
+documentation which covers the details for that specific repository. In most
+cases, the GitHub repository which contains the Dockerfiles for an Official
+Repository also has an active issue tracker. General feedback and support
+questions should be directed to `#docker-library` on [Libera.Chat IRC](https://libera.chat).
```

Comment:
Is it worth a dedicated channel, or should we just direct folks to `#general` (since they'll often post there anyhow, and their questions are not usually specific to DOI after all)?

As an example, with `##docker` on Libera.Chat being spiritually similar to `#general` on Slack, we don't get much _user_ chatter in `#docker-library` (mostly just maintainers looking for help, advice, or pre-review).

Reply:
I guess what I'm saying is that it might make more sense to point users to `#general` and make this note specific to maintainers?